lib/test/unit/ui/console/testrunner.rb

```diff
@@ -394,6 +395,12 @@ def output_statistics(elapsed_time)
             change_output_level(IMPORTANT_FAULTS_ONLY) do
               output("Finished in #{elapsed_time} seconds.")
             end
+            if @options[:report_slow_tests]
+              @slow_tests.sort_by { |slow_test| -slow_test[:elapsed_time] }
```

Comment:
Could you the following style?

```suggestion
              @slow_tests.sort_by {|slow_test| -slow_test[:elapsed_time]}
```

Reply:
OK!